src/test/java/seedu/address/model/person/TutorialGroupContainsKeywordsPredicateTest.java

```diff
@@ -45,30 +45,26 @@ public void equals() {
     public void test_tutorialGroupContainsKeywords_returnsTrue() {
         // One keyword
         TutorialGroupContainsKeywordsPredicate predicate =
-                new TutorialGroupContainsKeywordsPredicate(Collections.singletonList("912345"));
-        assertTrue(predicate.test(new PersonBuilder().withTutorialGroup("912345").build()));
+                new TutorialGroupContainsKeywordsPredicate(Collections.singletonList("T19"));
+        assertTrue(predicate.test(new PersonBuilder().withTutorialGroup("T19").build()));
 
         // Only one matching keyword
-        predicate = new TutorialGroupContainsKeywordsPredicate(Arrays.asList("9123456", "908976"));
-        assertTrue(predicate.test(new PersonBuilder().withTutorialGroup("908976").build()));
-
-        // Non-matching keyword
-        predicate = new TutorialGroupContainsKeywordsPredicate(Arrays.asList("908976", "9123456"));
-        assertTrue(predicate.test(new PersonBuilder().withTutorialGroup("9123456908976").build()));
```

Comment:
Perhaps you may consider if this is useful?
predicate = new TutorialGroupContainsKeywordsPredicate(Arrays.asList("T12", "T13"));
assertFalse(predicate.test(new PersonBuilder().withTutorialGroup("T12T13").build()));

Reply:
Hm I think `T12T13` will cause an exception to be thrown due to the TG format